src/Models/ProductList.tsx

```diff
@@ -1,7 +1,10 @@
 import * as React from 'react';
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
 
+import FiltersComponent from '@/components/Filters/Filters.component';
 import Pagination from '@/components/Pagination/Pagination.component';
+import type { CategoryEnum } from '@/pages/products/useFilters';
```

Comment:
<img width="462" alt="Screenshot 2024-06-18 at 12 20 08" src="https://github.com/igor0806/fe-react-2024/assets/102523692/83949236-6bb8-4a5b-b7b6-18057df2dfb8">
filters UI is broken